packages/expo-gl-cpp/cpp/EXGLContext.cpp

```diff
@@ -0,0 +1,93 @@
+#include "EXGLContext.h"
```

Comment:
In general in unimodules we use `<PodName/HeaderFile.h>` notation to also pinpoint the lookup in the appropriate spec. This allows us to include files from different pods.

Could we move to `<EXGL_CPP/EXGLContext.h>` import? Would it work on Android?

Reply:
maybe if we add EXGL_CPP inside cpp directory and set include path it would allow this on android but I'm not sure if it won't break anything on ios
Either way, I think this type of change should be done after jsi rewrite.

Reply:
since you already changed `""` import to `<>` in `EXiOSUtils.mm`, could you apply the change that Stanisław requested?